### PR TITLE
ppp

### PR DESCRIPTION
**Pull request by riotoff**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/linstall.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "riotoff"
+    },
```
Comment (CuteDog5695) [inline domains/linstall.json:4]: Please add another form of contact (Like Discord, Twitter etc.)

File: domains/linstall.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    "record": {
+        "CNAME": "riotoff.github.io"
```
Comment (CuteDog5695) [inline domains/linstall.json:6]: 404 page, please give a preview of your website.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/linstall.json (hunk 2 above)
Comment (riotoff) [reply domains/linstall.json]: https://riotoff.github.io/linstall

**Comment by CuteDog5695** [comment]
Didn't fix the social part.

**Comment by riotoff** [comment]
> Didn't fix the social part.

what i did to do?

**Comment by CuteDog5695** [comment]
> Please add another form of contact (Like Discord, Twitter etc.)

File: domains/linstall.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "riotoff"
+    },
+    "record": {
+        "CNAME": "riotoff.github.io/linstall"
```
Comment (CuteDog5695) [inline domains/linstall.json:6]: I FORGOT ABOUT THIS
```suggestion
        "CNAME": "riotoff.github.io"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]

**Pull request merged**